pytradfri/device/air_purifier.py

```diff
@@ -55,6 +60,31 @@ def raw(self) -> TypeAirPurifier:
             cast(TypeRawList, self.device.raw)[ROOT_AIR_PURIFIER][self.index],
         )
 
+    @property
+    def motor_runtime_total(self) -> int:
+        """Return runtime of fan motor, expressed in minutes."""
+        return self.raw[ATTR_AIR_PURIFIER_MOTOR_RUNTIME_TOTAL]
+
+    @property
+    def filter_lifetime_total(self) -> int:
+        """Return total lifetime of filter, expressed in minutes."""
+        return self.raw[ATTR_AIR_PURIFIER_FILTER_LIFETIME_TOTAL]
+
+    @property
+    def filter_status(self) -> int:
+        """Return filter status."""
```

Comment:
What does this integer represent in terms of filter status?

Reply:
I am not sure to be honest as I haven't had my filter for more than three months (its lifetime seems to be six months). If I were to make a guesss it will return `1` when its time to replace the filter. But I'm not willing to put that into the documentation of the property yet.

Reply:
Does it return 0 today for you?

Reply:
Yes:

```
{'5900': 1,
 '5902': 152245,
 '5903': 0,
 '5904': 259200,
 '5905': 0,
 '5906': 1,
 '5907': 8,
 '5908': 10,
 '5909': 152244,
 '5910': 106955,
 '9003': 0}
```